### PR TITLE
update the position

### PR DESCRIPTION
**Pull request by IrisK31**

### CODE REVIEW

File: content/about-cn/recruit/TOC.md [hunk 1]
```diff
@@ -14,6 +16,7 @@
   - [工程效率研发工程师](engineering/engineering-efficiency-engineer.md)
   - [运维工程师](engineering/sa-engineer.md)
 - Business
+  - [高级行业方案架构师](business/senior-solution-architect)
```
Comment (g1eny0ung) [inline content/about-cn/recruit/TOC.md:19]: Need to add `.md` extension in the end.

File: content/about-cn/recruit/TOC.md [hunk 2]
```diff
@@ -33,6 +36,7 @@
   - [Content Strategist](i18n/content-strategist.md)
   - [Demand Generation Specialist](i18n/demand-generation-specialist.md)
 - MKT
+  - [高级产品市场经理](market/senior-product-marketing-manager.md)
```
Comment (g1eny0ung) [inline content/about-cn/recruit/TOC.md:39]: This file missed?

File: content/about-cn/recruit/TOC.md [hunk 3]
```diff
@@ -5,6 +5,8 @@
 - Engineering
   - [分布式存储系统研发经理](engineering/tikv-engineer.md)
   - [分布式调度研发工程师](engineering/pd-engineer.md)
+  - [数据库内核开发-优化器](engineering/optimizer-engineer.md)
+  - [Real-Time Analytics 研发工程师](real-time-analytics-engineer.md )
```
Comment (g1eny0ung) [inline content/about-cn/recruit/TOC.md:9]: Need to add an `engineering/` prefix.

File: content/about-cn/recruit/TOC.md (hunk 2 above)
Comment (Soline324) [reply content/about-cn/recruit/TOC.md:39]: hasn't upload yet

**Comment by Soline324** [comment]
Please check other updates in https://github.com/pingcap/pingcap.github.io/pull/460
@g1eny0ung @YiniXu9506 @IrisK31

File: content/about-cn/recruit/TOC.md (hunk 2 above)
Comment (g1eny0ung) [reply content/about-cn/recruit/TOC.md:39]: ok

**Pull request merged**